Update SSL mode, all screenshots and instructions for latest DBeaver (#14733)
cockroachdb/docs: v22.1/dbeaver.md

## Step 1. Start DBeaver and connect to CockroachDB

> 1. Start DBeaver, and select **Database > New Connection** from the menu.  In the dialog that appears, select **CockroachDB** from the list.

Reviewer: ```suggestion
1. Start DBeaver, and select **Database > New Connection** from the menu. In the dialog that appears, select **CockroachDB** from the list.
```

    <img src="{{ 'images/v22.1/dbeaver-01-select-cockroachdb.png' | relative_url }}" alt="DBeaver - Select CockroachDB" style="border:1px solid #eee;max-width:100%" />

1. Click **Next**. The **Connection Settings** dialog displays.

> 1. In the **Database** field, enter **movr**.

Reviewer: Should "movr" actually be in code ticks, since it is technically a value that we are entering? Not sure if our style is settled on this.

> 1. In the **Database** field, enter **movr**.

Author: ```suggestion
1. In the **Database** field, enter `movr`.
```

    <img src="{{ 'images/v22.1/dbeaver-02-cockroachdb-connection-settings.png' | relative_url }}" alt="DBeaver - CockroachDB connection settings" style="border:1px solid #eee;max-width:100%" />

## Step 2. Update the connection settings

1. Click the **SSL** tab.

    <img src="{{ 'images/v22.1/dbeaver-03-ssl-tab.png' | relative_url }}" alt="DBeaver - SSL tab" style="border:1px solid #eee;max-width:100%" />

1. Check the **Use SSL** checkbox as shown, and fill in the text areas as follows:
    - **Root certificate**: Use the `ca.crt` file you generated for your secure cluster.

>     - **SSL certificate**: Use a client certificate generated from your cluster's root certificate.  For the root user, this will be named `client.root.crt`.  For additional security, you may want to create a new database user and client certificate just for use with DBeaver.

Reviewer: ```suggestion
    - **SSL certificate**: Use a client certificate generated from your cluster's root certificate. For the root user, this will be named `client.root.crt`. For additional security, you may want to create a new database user and client certificate just for use with DBeaver.
```

>     - **SSL certificate key**: Because DBeaver is a Java application, you will need to transform your key file to the `*.pk8` format using an [OpenSSL command](https://wiki.openssl.org/index.php/Command_Line_Utilities#pkcs8_.2F_pkcs5) like the one shown below.  Once you have created the file, enter its location here.  In this example, the filename is `client.root.pk8`.

Reviewer: ```suggestion
    - **SSL certificate key**: Because DBeaver is a Java application, you will need to transform your key file to the `*.pk8` format using an [OpenSSL command](https://wiki.openssl.org/index.php/Command_Line_Utilities#pkcs8_.2F_pkcs5) like the one shown in the following snippet. Once you have created the file, enter its location here. In this example, the filename is `client.root.pk8`.
```

        {% include_cached copy-clipboard.html %}
        ~~~ console
        $ openssl pkcs8 -topk8 -inform PEM -outform DER -in client.root.key -out client.root.pk8 -nocrypt
        ~~~

> 1. Select **verify-ca** or **verify-full** from the **SSL mode** dropdown.  There is no need to set the **SSL Factory**, you can let DBeaver use the default.

Reviewer: ```suggestion
1. Select **verify-ca** or **verify-full** from the **SSL mode** dropdown. There is no need to set the **SSL Factory**; you can let DBeaver use the default.
```

## Step 3. Test the connection settings

1. Click **Test Connection ...**. If you need a driver, the following dialog displays:

    <img src="{{ 'images/v22.1/dbeaver-06-download-driver.png' | relative_url }}" alt="DBeaver - download driver dialog" style="border:1px solid #eee;max-width:100%" />

1. Click **Download**.

    After the driver downloads, if the connection was successful, you will see a **Connected** dialog.

    <img src="{{ 'images/v22.1/dbeaver-04-connection-success-dialog.png' | relative_url }}" alt="DBeaver - connection success dialog" style="border:1px solid #eee;max-width:100%" />

1. Click **OK** to dismiss the dialog.

## Step 4. Start using DBeaver

> Expand the **movr** database node and navigate to the **rides** table.

Reviewer: Similarly to above, perhaps "movr" and "rides" should be in code formatting instead? I realize they can also be considered interface elements in this context.
